packages/gatsby-remark-copy-linked-files/src/__tests__/index.js

```diff
@@ -211,6 +211,31 @@ describe(`gatsby-remark-copy-linked-files`, () => {
       })
     })
 
+    it(`copies file to destinationDir when supplied (with pathPrefix)`, async () => {
+      const markdownAST = remark.parse(`![some absolute image](${imagePath})`)
+      const pathPrefix = `/blog`
+      const validDestinationDir = `path/to/dir`
+      const expectedNewPath = path.posix.join(
+        process.cwd(),
+        `public`,
+        validDestinationDir,
+        `/undefined-undefined.gif`
```

Comment:
why `undefined`?

Reply:
### Why undefined

I believe the reason it's `undefined` is because when this is run, there are no actual images for it to the filenames from.

`undefined-undefined`.gif represents `FILENAME`-`DIGEST`.gif

I'm not sure where/when the filename and digest are created but at the time of the test, these values don't exist in the hash. I'm still looking into this.

It originates [here](https://github.com/seryl/gatsby/blob/8a9ed65a8a484d23f454f85f4e8fdae6651e66cd/packages/gatsby-remark-copy-linked-files/src/index.js#L21) but I'm not sure when those values actually get set (I'm still looking around).

### For the added test itself

All we're doing here is taking the [markdownAST children](https://github.com/seryl/gatsby/blob/8a9ed65a8a484d23f454f85f4e8fdae6651e66cd/packages/gatsby-remark-copy-linked-files/src/__tests__/index.js#L20).

This is just validating that the additional `pathPrefix` is joined.

The existing test without the prefix is here: [#L239-L258](https://github.com/seryl/gatsby/blob/8a9ed65a8a484d23f454f85f4e8fdae6651e66cd/packages/gatsby-remark-copy-linked-files/src/__tests__/index.js#L239-L258)

I'm just adding an additional test to validate that the prefix exists when it's set.